_documentation/user/flags/seer.md

```diff
@@ -5,4 +5,14 @@ type: Good
 summary: See stealthed, cloaked and masquerading tanks as normal.
 ---
 
-...
+The Seer (SE) flag allows the player to see the tanks holding the [Stealth (ST)](../stealth/), [Cloaking (CL)](../cloaking/), or [Masquerade (MQ)](../masquerade/) flags as normal.
+
+Its power is therefore to nullify the effects of those three flags for the holder, rather than bringing out a unique power-up itself.
+
+## Tactics
+
+As the flag disables the effect of three other specific flags for the holder, the usage of Seer is highly situational.
+
+In general, a player would prefer a flag that gives a power-up itself rather than nullify that of some others, as a power-up can effect most players on the map, while Seer does not add any effect when battling against tanks that have flags other than the three affected by Seer.
```

Comment:
```suggestion
A player may prefer a flag that gives a power-up itself rather than nullify that of some others, as a power-up can effect most players on the map, while Seer does not add any effect when battling against tanks that have camouflaging flags.
```

No hard count and slightly more objective?

Reply:
why is a preference part of tactics? Focus on how to use to advantage, particularly finding a pesky stealth tank (think Rabbit Hunt)

Reply:
I'm just saying, "A player would (may) prefer" is a preference, not a tactic. All (non-bad) flags have some value. It seems to me the point is to describe the value.

Reply:
Then it can be worded as 'A player prefers' if it's such a sensitive wording, or perhaps swapping the word 'prefers' out for some other completely.

The point is to describe the tactics, not value, which seems, at least to me, when to and when not to use the flag (in relation to other flags).

I believe this line that you deem as preference is actually the most crucial one, as newer players can easily understand the effect, but not the tactics. They ask themselves 'I just grabbed this flag, should I keep it?'. And this line examines the situation of when it is a good tactic to keep it and, of course, when it is not.

And it isn't preference, it is effectiveness. Or, you could call it preference towards effectiveness, which is usually what one wants.

Reply:
@allejo @jwmelo in accordance to the post above, we ought to decide whether we will comprehend the 'tactics' field as flag 'value' (meaning positives only) or flag 'situational usage in relation to other flags' (meaning positives and negatives, or better worded, when to and when not to use a flag).

Reply:
@allejo @jwmelto in accordance to the post above, we ought to decide whether we will comprehend the 'tactics' field as flag 'value' (meaning positives only) or flag 'situational usage in relation to other flags' (meaning positives and negatives, or better worded, when to and when not to use a flag).

Reply:
Being the section is titled "Tactics" and has been standardized amongst all the flags, the section should be dedicated to actual tactics of the flag; i.e. how to use the flag and when or what strategies exist.

As for preferences/opinion, I agree that we shouldn't have that. We can have statements like this in the docs, but just the wording of it shouldn't imply opinions or preferences.

Reply:
I think we all agree on that.

Reply:
I don't want to go back and forth on this. Don't compare this flag to other flags, and why you would want another one over this one. Describe how to use this flag effectively. Period.

Reply:
Where did I ever compare this flag to another? If you are meaning to pose arguments, back them up. I wrote about this flag's strength is certain situations and weaknesses in others. I never mentioned any other specific flags, just all other flags as a collective group, which I believe is a matter very worth bringing up in the tactics field of this specific flag to examine its usage entirely.

Reply:
> In general, a player would prefer a flag that gives a power-up itself rather than nullify that of some others, as a power-up can effect most players on the map, while Seer does not add any effect when battling against tanks that have flags other than the three affected by Seer.

This does not describe tactics, but is explicitly a point of preference. It is, and has always been, the topic of this extensive thread. Just remove the sentence.

Reply:
Or even *less* mentioning of other flags:

> Seer does not add any effect when battling against tanks that have flags other than those affected by Seer.